librz/analysis/var.c

```diff
@@ -104,6 +105,7 @@ static void resolve_var_overlaps(RzAnalysisVar *var) {
 			rz_analysis_var_delete(other);
 		}
 	}
+	rz_pvector_free(cloned_vars);
```

Comment:
If var->fcn->vars is defined with a free function, this will free the original items of the vector as well, I think. Right @thestr4ng3r ?

Reply:
It doesn't, otherwise, it would fail on my tests. I am sure this code was called multiple times during the `aaa`/`aaft` tests

Reply:
I'm not 100% convinced... also, `RzPVector *cloned_vars = (RzPVector *)rz_vector_clone((RzVector *)&var->fcn->vars);` seems wrong, because rz_vector_clone returns a RzVector. Why does it need the cast? Why casting a PVector to a Vector?

Reply:
I copied the code from the one written by the @thestr4ng3r